proxy/envoy/agent.go

```diff
@@ -62,10 +66,14 @@ mixer_options {
   mixer_server: "%s%s"
 }
 	`, EgressClusterPrefix, mixer))
+	if err := f.Chmod(0777); err != nil {
```

Comment:
File creation and setting permissions can be done in one step, e.g. 
    os.OpenFile(path, O_CREATE | O_TRUNC | O_WRONLY, perms)